### ISSUE TITLE
Bots cannot use the flight powerup.

### ISSUE THREAD

**Issue by q3shafe** [q3shafe/trepidation#180]
Your right about the bots, bots never work with the flight powerup, an ideal fix would be to have them roam around while in flight, however the bot paths in maps have a lot to do with bot movement, a more short term fix would be to not allow them to pickup flight, I'll add a new issue for the bot part of this issue.

**Comment by q3shafe** [comment]
Need to test this

**Comment by q3shafe** [comment]
Has anyone noticed and/or been able to successfully see if my changes are working.. I'm having a hard time catching it...

**Comment by angrynerd** [comment]
I thought i saw a bot with the flight powerup on aversion 2 at one point
but i killed him from a distance and could replicate it so i dont know.

**Comment by angrynerd** [comment]
yup can confirm a bot was using the flight power up on area52. he was essentially stuck where he picked it up. i couldnt get a screen shot at the time but just know it can happen.
